webapp/channels/src/components/invitation_modal/invite_view.tsx

```diff
@@ -233,12 +227,8 @@ export default function InviteView(props: Props) {
                         props.onChangeUsersEmails(usersEmails);
                     }}
                     value={props.usersEmails}
-                    validAddressMessageId={props.inviteType === InviteType.MEMBER ? t(
-                        'invitation_modal.members.users_emails_input.valid_email',
-                    ) : t('invitation_modal.guests.users_emails_input.valid_email')}
-                    validAddressMessageDefault={props.inviteType === InviteType.MEMBER ? 'Invite **{email}** as a team member' : 'Invite **{email}** as a guest'}
-                    noMatchMessageId={noMatchMessageId}
-                    noMatchMessageDefault={noMatchMessageDefault}
+                    validAddressMessage={validAddressMessage}
+                    noMatchMessage={noMatchMessage}
```

Comment:
This is one of a few places where we went from passing in separate ID/default props and now we just pass in a single `MessageDescriptor` for each piece of text